python/pyspark/ml/classification.py

```diff
@@ -47,14 +47,41 @@
            'OneVsRest', 'OneVsRestModel']
 
 
+class JavaClassifierParams(HasRawPredictionCol, JavaPredictorParams):
+    """
+    (Private) Java Classifier Params for classification tasks.
+    """
+    pass
+
+
+@inherit_doc
+class JavaClassifier(JavaPredictor, JavaClassifierParams):
+    """
+    Java Classifier for classification tasks.
+    Classes are indexed {0, 1, ..., numClasses - 1}.
+    """
+
+    def setRawPredictionCol(self, value):
```

Comment:
can we add a since annotation?

Reply:
`HasRawPredictionCol` in `JavaClassifierParams` already support this method, right?
My previous PR https://github.com/apache/spark/pull/25662/files added this method because the setters generated by code_gen were removed in it.

Reply:
As a broad comment, I'd say retain `@Since` information if it already existed. Did this exist already? if it was added in 3.0 we can mark it as such. If it replaces several definitions in subclasses that have inconsistent "since" versions, hm, I think we should label it as "since" the latest version of any of them.

Reply:
@zhengruifeng Thanks for your comments. 
You are right, ```HasRawPredictionCol``` already has this ```setRawPredictionCol```. I am a little hesitated to remove this setter, because I don't want to have too many no-ops class in this PR.  Since we will need to add it back after removing the setters from code_gen, I guess maybe just leave the method here?

Reply:
I think it can stay as is